apps/storybook/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
Is this file needed?

Reply:
Tests apparently work without it, so removed